### DISCUSSION TITLE
q-expansion-item expanded item label highlight

### DISCUSSION THREAD

**Discussion by mytc** [quasarframework/quasar#8813] [General - Components / Directives / etc]
I like to know how to styling the q-expansion-item label. 
When there is the expanded item, how do I change the background of the label. 
Also, is there anyway to change the label font size? 
As far as I know, it does not allow to use html tag in Expansion Item label.

**Comment by zeynab-rs** [comment]
By `q-item__label` class, you can style it.

**Reply by mytc** [reply]
Nope, tried. Not working. 
Even so, it still does not solve the expanded item label styling problem.

**Reply by hawkeye64** [reply]
Did you make the css scoped? If so, it won't work unless you use the deep selector (look at Vue docs for more info).
